themes/default/content/blog/project-scoped-stacks-in-self-managed-backend/index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Projects in Self-Managed State Backends"
+date: 2023-03-29
+meta_desc: Pulumi's Self-Managed backend now supports stack names scoped by project name.
+meta_image: meta.png
+authors:
+    - justin-vanpatten
+    - abhinav-gupta
+tags:
+    - features
+    - pulumi-releases
+---
+
+At Pulumi, we aim to offer our users the best-in-class IaaC experience.
```

Comment:
```suggestion
At Pulumi, we aim to offer our users the best-in-class IaC experience.
```

Reply:
🤦  Yes. My brain and fingers disagreed about what I was writing.